R/IOHanalyzer.R

```diff
@@ -18,29 +18,124 @@
 #' @useDynLib IOHanalyzer
 NULL
 #Ugly hack, but appears to be required to appease CRAN
-utils::globalVariables(c(".","algId","run","ERT","RT","max_samples",
+utils::globalVariables(c(".", "algId", "run", "ERT", "RT", "group",
                          "DIM", "Fvalue", "lower", "upper", "target", "format",
-                         "runtime", "parId", "instance", "input", "funcId"))
+                         "runtime", "parId", "instance", "input", "funcId",
+                         "budget", "dimension", "loss", "name", "optimizer_name",
+                         "rescale"))
 
 options(shiny.port = 4242)
 
-probs <- c(2, 5, 10, 25, 50, 75, 90, 95, 98) / 100.
+.onLoad <- function(libname, pkgname) {
```

Comment:
.onLoad will be called when the user loads the package?

Reply:
Yes, this will be called with every 'library(IOHanalyzer)' call